### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by davideme**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,10 +60,13 @@ class SampleViewController: BothamViewController, SampleUI {
     /*...*/
 }
 ```
+### Dependency injection
 
-#### Service Locator
+BothamUI is build around the concept of dependency injection, all the dependencies are provided by constructor or properties, base what UIKit allow us.
```
Comment (Serchinastico) [inline README.md]: - BothamUI is buil**t**
- UIKit allow**s** us

Comment (pedrovgs) [inline README.md]: Base what? or base that?

File: README.md [hunk 2]
```diff
@@ -60,10 +60,13 @@ class SampleViewController: BothamViewController, SampleUI {
...
 
-We recommend the usage of a Service Locator pattern in order to instantiate view controllers.
+#### ViewController Instantiation
+
+In the example a Service Locator in order to instantiate view controllers, but you can also use [Swinject](https://github.com/Swinject/Swinject) or others DI frameworks.
```
Comment (pedrovgs) [inline README.md]: In the example a Service Locator **is used**

**Comment by pedrovgs** [comment]
:+1:

**Comment by Serchinastico** [comment]
:+1:

**Pull request merged**